HelpSource/Classes/UnitTest.schelp

```diff
@@ -229,9 +278,7 @@ TestYourClass : UnitTest {
 	}
 
 	test_yourMethod {
-
-		// every method named test_
-		// will be run
+		// every method named test_something  will be run
```

Comment:
I think this would be clearer as "whose name begins with `test_`". either way, can you remove the extra space here?